packages/kbn-reporting/public/share/share_context_menu/register_pdf_png_modal_reporting.tsx

```diff
@@ -43,154 +40,8 @@ const getJobParams = (opts: JobParamsProviderOptions, type: 'pngV2' | 'printable
 };
 
 /**
- * This is used by Canvas
+ * This is used by Dashboard and Visualize apps (sharing modal)
  */
-export const reportingScreenshotShareProvider = ({
```

Comment:
The Canvas integration doesn't use this export. See `x-pack/plugins/canvas/public/components/workpad_header/share_menu/share_menu.tsx`